pages/view/engagement/_id.vue

```diff
@@ -19,7 +19,7 @@
     <div class="flex mb-4">
       <div class="w-5/12 margins">
         <EngViewFields label="date">
-          {{ engagement.date }}
+          {{ engagement.date.substring(0, 10) }}
```

Comment:
I get one small bug with this line where sometimes the object or the date property is undefined and I get this error:

![Screen Shot 2020-07-14 at 1 44 48 PM](https://user-images.githubusercontent.com/31868510/87458579-30c44b80-c5d8-11ea-9a9c-d097d133777b.png)

This only ever happens on the first selection of an EngagementListItem after a page refresh from the Engagements Search view.

Reply:
I fixed it by changing the fetch() hook to asyncData(), you can check it out at 235-date-format-jw. If you want just merge it into your branch if it works.